docs/pages/enterprise/sso.mdx

```diff
@@ -148,9 +148,9 @@ values to match your identity provider:
   to the full [ADFS guide](./sso/adfs.mdx#create-teleport-roles) for details.
 - `netiq` (OIDC): Used to enable NetIQ-specific ACR value processing; refer to
   the [OIDC guide](./sso/oidc.mdx#optional-acr-values) for details.
-- `ping` (SAML): Required for compatibility with Ping Identity (including
-  PingOne and PingFederate) when "Enforced Signed Authn Requests" is enabled in
-  the application settings.
+- `ping` (SAML and OIDC): Required for compatibility with Ping Identity (including
+  PingOne and PingFederate). For SAML compatibility when "Enforced Signed Authn Requests" is enabled in
```

Comment:
What would you think of removing the text after `Required for compatibility with Ping Identity (including PingOne and PingFederate).`? I'm not sure it would make a difference for this config option whether a Ping user has selected "Enforced Signed Authn Requests" or is using the client secret authentication method. Either way, the value of this setting is `ping`.

If the goal of this PR is to explain to Ping Identity users how they can enable the OIDC or SAML connector from the Ping Identity side, I think we should expand this information and put it elsewhere, such as a short guide to using Ping Identity to complement our guides for GitLab, Okta, etc.

Reply:
Done @ptgott